app-modules/archeo/src/Console/Commands/ImportExternalPdfsCommand.php

```
@@ -0,0 +1,130 @@
+
+        $files = [];
+        if (is_dir($path)) {
+            $files = glob(rtrim($path, DIRECTORY_SEPARATOR).DIRECTORY_SEPARATOR.'*.{pdf,PDF}', GLOB_BRACE);
+        } elseif (is_file($path)) {
+            if (strtolower(pathinfo($path, PATHINFO_EXTENSION)) === 'pdf') {
+                $files = [$path];
+            } else {
+                $this->error("The file [{$path}] is not a PDF.");
+
+                return 1;
+            }
```

PHPMD (2.15.0)

[error] The method handle uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)